### ISSUE TITLE
Product recommendations

### ISSUE THREAD

**Issue by forsbergplustwo** [davidcelis/recommendable#116]
Hey David,

Firstly, apologies that this is a question and not an issue, per se.

Thanks for the gem and the very informative, and humurous, article explaining the concepts. Well worth a tweet!

I'm looking to build a very simple product recommendation engine, where by buying a product the user 'likes' a product.

I would then, for other users, recommend products based on their purchased/liked product similarities.

Basically including the recommendation for other products AFTER they have purchased something themselves.. in follow up emails to be exact.

From my very basic understanding, is this an ok way to go about things?

My thinking is that I can build up recommendations over time, and have a good way to auto-recommend based on similar customer purchases compared to your own purchase.

Many thanks,
Bjorn
